guides/common/modules/con_security-features.adoc

```diff
@@ -0,0 +1,43 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-04-23
+:_mod-docs-content-type: CONCEPT
+
```

Comment:
```suggestion
```
Pls, remove.
FYI, there's the `-D` option in newdoc to avoid this metadata header.